### PR TITLE
Added notification for youtube live stream

### PR DESCRIPTION
**Pull request by therealjuan**
1. Pull the branch or run the demo service
2. Visit /homepage or /store to see a notification on the live stream happening Tonight
3. #winning

### CODE REVIEW

**Comment by evandandrea** [comment]
Given your intro, I'm assuming it's intention that this doesn't show on /blog or /iot. I ran locally and it looks good.

File: webapp/template_utils.py [hunk 1]
```diff
@@ -95,3 +95,29 @@ def display_name(display_name, username):
         return display_name
     else:
         return f"{display_name} ({username})"
+
+
+def time_live_stream():
+    """Temporary function
+    returns: 0 if stream is before the time
+    returns: 1 if stream is now
+    returns: 2 uf stream is close
```
Comment (evandandrea) [inline webapp/template_utils.py]: Typo.

File: webapp/template_utils.py [hunk 2]
```diff
@@ -95,3 +95,29 @@ def display_name(display_name, username):
...
+    stream = datetime.datetime(year=2019, month=4, day=17, hour=20, tzinfo=tz)
+
+    end_stream = datetime.datetime(
+        year=2019, month=4, day=17, hour=22, tzinfo=tz
+    )
+
+    if now < stream:
+        return 0
+    elif end_stream < now:
+        return 2
```
Comment (evandandrea) [inline webapp/template_utils.py]: Why? This doesn't get used.

Comment (tbille) [reply webapp/template_utils.py]: yes once the strem is done

**Pull request merged**